src/box/lua/misc.cc

```diff
@@ -502,7 +466,7 @@ box_lua_misc_init(struct lua_State *L)
 	static const struct luaL_Reg boxlib_internal[] = {
 		{"prepare_auth", lbox_prepare_auth},
 		{"select", lbox_select},
-		{"new_tuple_format", lbox_tuple_format_new},
+		{"new_external_tuple_format", lbox_new_external_tuple_format},
```

Comment:
Why all the format related functions are moved, but not this one?

Reply:
Because it is used by `net.box` and it has a different logic (i.e., only a tuple dictionary is created).

Reply:
Personally I'm still looking on it as on one of possible tuple format constructors, so it feels natural to have it together with other tuple format specific functions.

I have updated the discussion right above (regarding the name of the function) with some thoughts around.

I'll leave you to decide, please, take it as just my thoughts.